### PR TITLE
added description value with key of item type

### PR DESCRIPTION
**Pull request by nabeelamirch**
Added description value with key of item type.

resolves: GCI-962

### CODE REVIEW

**Comment by VaughanJackson123** [comment]
Not sure what is happening here. It is showing me that changes have been made to the file `filing_history_descriptions.yml`, but I can't see what they are.
Nonetheless, I am surprised this file has changed (if it has). Do we need a change to `orders_descriptions.yaml` maybe?

**Comment by nabeelamirch** [comment]
@VaughanJackson123 I have not modified the api-enumerations in anyway, I have no idea why it says it has. Maybe it was updated recently and we haven't merged changes from it?

File: src/test/java/uk/gov/companieshouse/items/orders/api/service/DescriptionProviderServiceTest.java [hunk 1]
```diff
@@ -17,11 +18,15 @@
 class DescriptionProviderServiceTest {
 
     private static final String COMPANY_NUMBER = "00006400";
+    private static final String ITEM_TYPE = "certificate";
```
Comment (VaughanJackson123) [inline src/test/java/uk/gov/companieshouse/items/orders/api/service/DescriptionProviderServiceTest.java:21]: This has been opened up now to take other item types. Have we tested it with those other item types? Is that possible at this stage?

Comment (nabeelamirch) [reply src/test/java/uk/gov/companieshouse/items/orders/api/service/DescriptionProviderServiceTest.java:21]: We would have to test the description of each item. I've had a look and it appears we would have to add descriptions for scud and certificated copies in orders_description.yml? And we don't know this yet?

Comment (VaughanJackson123) [reply src/test/java/uk/gov/companieshouse/items/orders/api/service/DescriptionProviderServiceTest.java:21]: Hence my earlier question about that descriptions file. Fine, if we don't know what those descriptions are then we can't do it! :)

**Pull request merged**